### DISCUSSION TITLE
Not trusted CA error when installing agent

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#10529] [2.4]
Im getting not trusted CA when installing the agent using the downloaded package from SOC.
Is there an easy way to extract the CA cert from SOC to deploy it elsewhere.

Or maybe ship the CA cert with the packages, and install with the agent binary

**Comment by defensivedepth** [comment]
Does this look like your issue:

https://github.com/Security-Onion-Solutions/securityonion/discussions/10504

**Reply by OzzyKampha** [reply]
Im one step later than that. After the agent has been enrolled to the fleet, the beats used the same cert to send data. if the cert is not trusted, the beats will fail because of it
